plugins/filter/vault_login_token.yml

```diff
@@ -0,0 +1,79 @@
+---
+DOCUMENTATION:
+  name: vault_login_token
+  short_description: Extracts the Vault token from a login or token creation
+  version_added: 2.2.0
+  description:
+    - Extracts the token value from the structure returned by a Vault token creation operation.
+  seealso:
+    - module: community.hashi_vault.vault_login
+    - module: community.hashi_vault.vault_token_create
+    - ref: community.hashi_vault.vault_login lookup <ansible_collections.community.hashi_vault.vault_login_lookup>
+      description: The official documentation for the C(community.hashi_vault.vault_login) lookup plugin.
+    - ref: community.hashi_vault.vault_token_create lookup <ansible_collections.community.hashi_vault.vault_token_create_lookup>
+      description: The official documentation for the C(community.hashi_vault.vault_token_create) lookup plugin.
+    - ref: Filter Guide <ansible_collections.community.hashi_vault.docsite.filter_guide.vault_login_token>
+      description: The C(community.hashi_vault) Filter Guide
+  notes:
+    - >-
+      This filter is the equivalent of reading into the dictionary directly,
+      but it has the advantages of providing semantic meaning
+      and automatically working against the differing output of modules and the lookups.
+  options:
+    _input:
+      description:
+        - A dictionary matching the structure returned by a login or token creation.
+      type: dict
+      required: true
+    optional_field:
+      description:
+        - >-
+          If this field exists in the input dictionary, then the value of that field is taken the be the login response,
+          rather than the input dictionary itself.
+        - >-
+          The purpose of this is primarily to deal with the difference between the output of lookup plugins
+          (which return the login response directly) and modules,
+          which return the login response in a C(login) field in its return.
+        - >-
+          This can be changed in case you've put the raw login response in some other structure,
```

Comment:
Here I lose the plot a bit, but rephrasing it as either "You can do X to accomplish Y" or "If you do X, Ansible does Y" should make this easier to read and translate.